src/tir/schedule/primitive/cache_read_write.cc

```diff
@@ -160,6 +160,21 @@ Block MakeCacheStage(const BufferRegion& cache_region, CacheStageInfo* info,
   return block;
 }
 
+/*!
+ * \brief Recalculate the `affine_binding` flag of the scope block info.
+ * \param block_sref The sref to the interested scope block.
+ */
+bool CalculateAffineFlag(const ScheduleState& self, const StmtSRef& block_sref) {
+  if (block_sref->parent == nullptr) {
+    return true;
+  }
```

Comment:
I have a question for a long period of time: if a block has no `IterVar` (i.e., opaque block), does it have affine bindings?

Reply:
I don't actually have a good question. CC: @Hzfengsy

Reply:
That's a good question. I think `affine flag` for opaque block makes no sense to me, no matter it's affine or not.
Blocks with affine bindings have more schedule abilities than other blocks, while opaque blocks have no schedule abilities.